introduction.md

```diff
@@ -1,4 +1,9 @@
 ## Introduction
 
 My name is Pavel Pontryagin. I'm studying GitHub.
-My favourite path is JavaScript path.
\ No newline at end of file
+My favourite path is JavaScript path.
+
+Skills:
+* Git
+* Java
+* Cassandra

```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?